pkg/controller/selector/filter_test.go

```diff
@@ -379,3 +380,51 @@ func TestDiskTypeFilter(t *testing.T) {
 		}
 	}
 }
+
+func TestGetPoolCapabilityMap(t *testing.T) {
+	Pool := model.StoragePoolSpec{
+		BaseModel: &model.BaseModel{
+			Id:        "f4486139-78d5-462d-a7b9-fdaf6c797e11",
+			CreatedAt: "2017-10-24T15:04:05",
+		},
+		FreeCapacity:     int64(50),
+		AvailabilityZone: "az1",
+		Extras: model.ExtraSpec{
+			"thin":     true,
+			"dedupe":   true,
+			"compress": true,
+			"diskType": "SSD",
+		},
+	}
+
+	var mapA map[string]interface{}
+	mapA = make(map[string]interface{})
+	mapA["key1"] = "value1"
+	mapA["key2"] = "value2"
+	Pool.Extras["mapA"] = mapA
+
+	result, err := GetPoolCapabilityMap(&Pool)
+	if nil != err {
+		t.Errorf("Expected nil, get %v", result)
+	}
+
+	CreatedAt, ok := result["createdAt"].(string)
+	if (!ok) || (Pool.CreatedAt != CreatedAt) {
+		t.Errorf("Expected true/%v, get %v/%v ", Pool.CreatedAt, ok, CreatedAt)
```

Comment:
What do you mean in `/%v`?

Reply:
I compare 2 values, so separate them with /